src/js/index.js

```diff
@@ -1 +1,44 @@
 import "../styles/main.scss";
+
+const $ = (selector) => document.querySelector(selector)
+
+const emailRegex = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+
+$(".js-email-input").addEventListener("keyup", (e) => {
+    let email = e.target.value
+    if (!emailRegex.test(email)) {
+        $(".js-error").textContent = "Please enter a valid email"
+        $(".js-submit-button").disabled = true
+    }else {
+        $(".js-error").textContent = ""
+        $(".js-submit-button").disabled = false
+    }
+})
+$(".js-form").addEventListener("submit", (e) => e.preventDefault())
+
+$(".js-submit-button").addEventListener("click", () => {
+    let email = $(".js-email-input").value
+    if (!emailRegex.test(email)) {
+        $(".js-error").textContent = "Please enter a valid email"
+        $(".js-submit-button").disabled = true
+    }else {
+        $(".js-error").textContent = ""
+        $(".js-submit-button").disabled = false
+        let httpRequest
```

Comment:
If this is working and tested we can leave it but if you want something a little easier to maintain, i'd be happy going with an AJAX library here. (Axios of fetch)

Reply:
i'll leave this up to you

Reply:
And if we **do** use jQuery, then use the jQuery AJAX functions - once the library is loaded, might as well use it well.

Reply:
We should be using deffered API if we do use jquery.

Reply:
I'd just use `fetch`, to be honest. Assuming this is all being transpiled anyway, it's by far the easiest option.

Reply:
that gets my vote 👍  fetch API it is.

Reply:
Dont forget to add a polyfill for fetch